packages/botonic-react/src/components/whatsapp-button-list.tsx

```diff
@@ -0,0 +1,67 @@
+import { INPUT } from '@botonic/core'
+import React from 'react'
+
+import { renderComponent } from '../util/react'
+import { Message } from './message'
+
+// TODO: Add validation in component
+
+interface Action {
+  button: string
+  sections: Section[]
+}
+
+interface Section {
+  title: string
+  rows: Row[]
+}
+
+interface Row {
+  id: string
+  title: string
+  description: string
+}
+
+const serialize = _whatsappTemplateProps => {
+  // TODO: Implement to have data persistance in localStorage, not needed for this WhatsApp development
+  return {}
+}
+
+export interface WhatsappButtonListProps {
+  header: string
+  body: string
+  footer: string
+  action: Action
```

Comment:
Why is the header and the footer props needed?

Reply:
they should be optional